barter-data/src/exchange/coinbase/book/l2.rs

```diff
@@ -0,0 +1,665 @@
+use std::fmt::Debug;
```

Comment:
Sorry, it's not well documented, but could you run `cargo +nightly fmt`? It will aggregate imports etc